src/components/common/style.ts

```diff
@@ -5,6 +5,7 @@ export const decodeItem: CSSProperties = {
   padding: '0.8rem 1rem',
   overflowY: 'auto',
   minHeight: '80px',
+  width: '100%',
```

Comment:
```suggestion
  width: '100%',
  boxSizing: 'border-box',
```

boxSizing 옵션이 누락되어서 100% 로 맞추면 padding으로 인해 영역박으로 overflow됩니다.
보조해주면 좋을것 같아요.